docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -777,17 +777,28 @@ Each treatment category has a hazard ratio associated with it both for progressi
      - 1.331 (1.324, 1.337)
      - 1.181 (1.171, 1.190)
 
-A lognormal distribution of uncertainty within the uncertainty intervals reported above should be assumed. The code block below instructs how to construct a distribution for each hazard ratio so that it can be sampled from.
+.. _`sampling instructions`:
 
-.. code-block:: python
+A lognormal distribution of uncertainty within the uncertainty intervals reported above should be assumed. Additionally, each time a value for a progression free survival hazard ratio is sampled for a given parameter (including each treatment, covariate, and risk factor effect), the same percentile within the distribution of uncertainty should be sampled for overall survival hazard ratio for that parameter. This dependent sampling strategy was chosen because PFS and OS hazard ratios are not independent of each other given that PFS is inclusive of OS.
+
+The code block below instructs how to construct a distribution for each hazard ratio so that it can be sampled from.
 
-	from numpy import log
-	from scipy.stats import norm, lognorm
+.. code-block:: python
 
-	q_975_stdnorm = norm().ppf(0.975)
-	mu = log(hr)
-	sigma = (log(hr_upper) - mu) / q_975_stdnorm
-	hr_distribution = lognorm(s=sigma, scale=hr)
+  from numpy import log
+  from scipy.stats import norm, lognorm
+  import random
+
+  def sample_paired_pfs_and_os_hazard_ratio_values(pfs_mid, 
+                                                   pfs_upper, 
+                                                   os_mid, 
+                                                   os_upper, 
+                                                   seed):
+    random.seed(seed)
+    pfs_value = lognorm(s=(log(pfs_upper) - log(pfs_mid)) / q_975_stdnorm, scale=pfs_mid).ppf(random.random())
```

Comment:
Are we supposed to set mu (scale) = log(hazard_mid) in a log-normal distribution?

Reply:
I'm not quite sure what you mean. I think this is the correct way to sample from a lognormal distribution. Do you disagree?

This is code that we have used in the past that I simplified and it produces the expected values for the mean/median and 97.5/2.5 percentiles

```
from numpy import log
from scipy.stats import norm, lognorm

# median and 0.975-quantile of lognormal distribution for RR
median = 2.22
q_975 = 5.26

# 0.975-quantile of standard normal distribution (=1.96, approximately)
q_975_stdnorm = norm().ppf(0.975)

mu = log(median) # mean of normal distribution for log(RR)
sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)

# Frozen lognormal distribution for RR, representing uncertainty in our effect size
# (s is the shape parameter)
rr_distribution = lognorm(s=sigma, scale=median)
```

Reply:
My question is about the use of mean versus median to calculate standard deviation.